Swiftfin tvOS/Views/SelectUserView/SelectUserView.swift

```diff
@@ -342,19 +350,58 @@ struct SelectUserView: View {
             serverSelection = .server(id: server.id)
         }
         .onNotification(.didDeleteServer) { server in
-            viewModel.send(.getServers)
+            didDelete(server)
+        }
+        .alert(
```

Comment:
Let's switch this out for `confirmationDialog` to mirror some of the other deletion presentation we have. This is my best tvOS example:

https://github.com/jellyfin/Swiftfin/blob/main/Swiftfin%20tvOS/Views/ItemView/Components/ActionButtons/ActionButtonHStack.swift#L125